styles.css

```diff
@@ -494,3 +499,403 @@ header nav a {
   right: 0;
   bottom: 40px;
 }
+
+@media (min-width: 768px) {
+  main {
+    background-image: url("./assets/images/header_desktop.png");
+    background-repeat: no-repeat;
+    background-size: 996px 822px;
+    height: 95vh;
+    margin-top: -109px;
+  }
+
+  header a {
+    margin: 8px;
+    font-size: 1.5rem;
+  }
+
+  header nav a {
+    display: none;
+  }
+
+  header nav ul {
+    display: flex;
+    margin-right: 80px;
+    padding: 10px;
+    font-style: normal;
+    font-weight: 500;
+  }
+
+  header nav ul li {
+    padding: 0 10px;
+  }
+
+  .flex {
+    position: relative;
+  }
+
+  .flex ul {
+    position: absolute;
+    top: 380px;
+    left: 37%;
+  }
+
+  .flex div {
+    position: absolute;
+    top: 90px;
+    left: 20px;
+  }
+
+  .flex p {
+    color: #172b4d;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    width: 70%;
+    margin: 0 auto;
+    padding: 0 10px;
+  }
+
+  .flex h1 {
+    font-size: 40px;
+    margin: 50px 0 40px 0;
+    padding-top: 0;
+  }
+
+  .projects {
+    margin: 0 20px;
+  }
+
+  .project-flex h2 {
+    font-size: 2.5rem;
+    padding: 20px 0;
+  }
+
+  .project-flex span {
+    display: none;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-column-gap: 8px;
+  }
+
+  .about-me {
+    background-image: url('./assets/images/about-lower.png');
+    background-size: 300px;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 350px;
+  }
+
+  .about-me h4 {
+    font-size: 2.9rem;
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .about-me button {
+    width: 20%;
+    margin-left: 40%;
+  }
+
+  .about-me p {
+    font-size: 1.2rem;
+  }
+
+  .card-sec {
+    border: 1px solid #c1c7d0;
+    background: #ebf0ee;
+    box-sizing: border-box;
+    border-radius: 8px;
+    width: 90%;
+    height: 320px;
+    margin: 20px 20px;
+  }
+
+  .card-section {
+    grid-column: 1/5;
+    margin-left: 40px;
+  }
+
+  .card-section1 {
+    grid-column: 5/9;
+  }
+
+  .card-section2 {
+    grid-column: 2/8;
+    margin-left: 70px;
+  }
+
+  .grid-ul {
+    display: grid;
+    grid-template-columns: repeat(9, 1fr);
+  }
+
+  .get-intouch h4 {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  .get-intouch form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin: 0;
+  }
+
+  .get-intouch input {
+    width: 80%;
+  }
+
+  .get-intouch textarea {
+    width: 80%;
+  }
+}
+
+@media (min-width: 1024px) {
+  main {
+    background-image: url("./assets/images/header_desktop.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    height: 95vh;
+    margin-top: -62px;
+  }
```

Comment:
- [ ] Kindly your background is not in the right position, you can use `background-size: cover;` instead of `background-size: contain;` it will solve the issue. To be like the Figma design.

![image](https://user-images.githubusercontent.com/35535130/155409041-3f18b1a4-e5fb-4096-873b-c69b680611a3.png)